week-3/Homework/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,22 @@
 
 var arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+arr.forEach(function (name) {
+  let newName = name;
+  if (name % 3 === 0) {
+    newName = "Fizz";
+  }
+
+  if (name % 5 === 0) {
+    newName = "Buzz";
+  }
+
+  if (name % 3 === 0 && name % 5 === 0) {
+    newName = "FizzBuzz";
+  }
+
+  console.log(newName);
+});
 /* EXPECTED OUTPUT */
```

Comment:
Good. This is quite an interesting problem, since its quite simple but there are lots of different ways of doing it. It's quite often used as a technical question in job interviews.

Reply:
Thanks for you feedback. I will find different ways of solving it. It is really interesting task, I enjoyed it.